stable_audio_tools/data/olac.py

```
@@ -0,0 +1,61 @@
+        sr: sample rate (int)
+    """
+    b = olac2flac(file.read())
+    fb = io.BytesIO(b)
+    y, sr = soundfile.read(fb)
+    return y, sr
+
+
+if __name__ == "__main__":
+    # test decoding
+    import sys
+    import math
```

Ruff (0.8.2)

`math` imported but unused

Remove unused import: `math`

(F401)